### ISSUE TITLE
'devices' command in CLI only reports one item on 0.5.0, reported all devices on 0.4.0

### ISSUE THREAD

**Issue by cs12ag** [benleb/surepy#11]
Not a priority issue, but there might be an problem with the 'devices' command in the CLI on 0.5.0. Here is the output from 0.5.0:
```
bash-5.0# surepy devices
         ╷           ╷                  ╷          ╷
      ID │ Household │ Name             │ Type     │ Serial
╶────────┼───────────┼──────────────────┼──────────┼────────╴
  nnnnnn │     nnnnn │ Backdoor Catflap │ Pet Flap │ None
         ╵           ╵                  ╵          ╵
```
and here is the output from 0.4.0:
```
bash-5.0# surepy devices

 surepy 🐾 v0.4.0

                            · Devices ·
         ╷           ╷          ╷                  ╷
      ID │ Household │ Type     │ Name             │       Serial
╶──────────────────────────────────────────────────────────────────
   nnnnn │     nnnnn │ Hub      │ Benbie's Hub     │ H002-nnnnnnn
  nnnnnn │     nnnnn │ Pet Flap │ Backdoor Catflap │            -
  nnnnnn │     nnnnn │ Feeder   │ Zamira’s Feeder  │ U003-nnnnnnn
  nnnnnn │     nnnnn │ Feeder   │ Benbie’s Feeder  │ U003-nnnnnnn
         ╵           ╵          ╵                  ╵
```

I do not have any Python knowledge, but I note there have been changes to the 'devices' command in the cli.py file recently. It would appear that the list of devices returned by SurePetcare() isn't being looped through correctly.

**Comment by benleb** [comment]
thanks! will check this

**Comment by jonootto** [comment]
Similar issue here, can only see the one flap, not the feeder or hub.

**Comment by benleb** [comment]
If you want, you can check out the dev branch :) If not, i plan to release the new (fixed) version next weekend :)

**Comment by benleb** [comment]
should be fixed in [0.6.0](https://github.com/benleb/surepy/releases/tag/v0.6.0)

if not, tell me :)
